sql-plugin/src/main/301db/scala/com/nvidia/spark/rapids/shims/spark301db/RapidsShuffleInternalManager.scala

```diff
@@ -1,78 +0,0 @@
-/*
- * Copyright (c) 2021, NVIDIA CORPORATION.
```

Comment:
why was this removed, I don't see one that replaces iit but maybe I'm missing it?